BAZProject/Views/MainViews/NewMovies/CollectionViewCells/NowPlayingCollectionViewCell.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  NowPlayingCollectionViewCell.swift
+//  BAZProject
+//
+//  Created by Mario Arceo on 01/03/23.
+//
+import UIKit
+
+class NowPlayingCollectionViewCell: UICollectionViewCell {
+    
+    let identifier = "NowPlayingCustomCell"
```

Comment:
Para no crear una instancia de esta clase es mejor poner el identifier con constante estática.
```suggestion
    static let identifier = "NowPlayingCustomCell"
```